pkg/tenant/api/v1/api.go

```diff
@@ -324,7 +318,6 @@ type APIKey struct {
 	Owner        string   `json:"owner,omitempty"`
 	Permissions  []string `json:"permissions,omitempty"`
 	Created      string   `json:"created,omitempty"`
-	Modified     string   `json:"modified,omitempty"`
```

Comment:
I think this is the only one I'm okay with removing. API keys are kind of a special case because they can't be edited. I believe every other resource can be edited, and we want to represent when that happened on the frontend.

Reply:
@pdeziel The story originated from the following comment when timestamps were added to the `Member` struct : https://github.com/rotationalio/ensign/pull/359#discussion_r1153615902

The `Modified` timestamp isn't used in Beacon, so I thought removing from the other structs as well would work best as something like `LastUsed` or `LastActivity` would likely be preferred as it is in the `Member` struct. We can do whichever is preferred.

The comment linked to above mentions removing the `Created` timestamp from the `Member` struct as well and that could also be done since we're using `DateAdded`.

Reply:
Gotcha, we can definitely rename the timestamps if that makes it easier for the frontend.